scss/components/_off-canvas.scss

```diff
@@ -154,10 +185,20 @@ $maincontent-class: 'off-canvas-content' !default;
 
     transform: translateX(-$size);
     overflow-y: auto;
+    
+    // Sets the position for nested off-canvas element
+    @at-root .#{$maincontent-class} .off-canvas.position-#{$position} {
```

Comment:
I was facing a lot specificity problems when mixing push & overlap with the nested feature. 
But since the push support for nested elements is removed (for now) I guess this may get simplified a bit.

Would you like me to work over it and see what I can cut down for this limited version?